experimental/datacommons-js/example.html

```diff
@@ -128,6 +128,20 @@ <h2>Map Web Component</h2>
         statVarName="Population Below Poverty Level Status in Past Year"
         statVarDcid="Count_Person_BelowPovertyLevelInThePast12Months"
       ></datacommons-map>
+      <h2>Ranking Web Component</h2>
+      <datacommons-ranking
+        title="US States with the Highest Population"
+        placeDcid="country/USA"
+        enclosedPlaceType="State"
+        variableDcid="Count_Person"
```

Comment:
Should we be consistent with the field names:  statVarDcid vs. variableDcid?

I slightly like variableDcid more.

Reply:
Acked -- Decision was already made to switch to variableDcid. Will use variableDcid from here on out, and will send out a followup PR to s/statVar/variable in the map web component.